app/models/error_report.rb

```diff
@@ -83,7 +83,7 @@ def cache_attributes_on_problem
   # Send email notification if needed
   def email_notification
     return false unless app.emailable?
-    return false unless app.email_at_notices.include?(@problem.notices_count)
+    return false unless app.email_at_notices.include?(0) || app.email_at_notices.include?(@problem.notices_count)
```

Comment:
I prefer to keep these guard clauses without compound conditionals. Could you put this on two lines?